### PR TITLE
Feature/september 2019 update

### LINKED ISSUE #31: Security vulnerability alert
Please review/advise:

1 lodash vulnerability found in …/webui/package.json 21 hours ago 
Remediation
Upgrade lodash to version 4.17.13 or later. For example: 
"dependencies": {
  "lodash": ">=4.17.13"
}
or…
"devDependencies": {
  "lodash": ">=4.17.13"
}
Always verify the validity and compatibility of suggestions with your codebase. 

Details
CVE-2019-10744 
More information
critical severity 
Vulnerable versions: < 4.17.13 
Patched version: 4.17.13 
Affected versions of lodash are vulnerable to Prototype Pollution.
The function defaultsDeep could be tricked into adding or modifying properties of Object.prototype using a constructor payload.

### PR DESCRIPTION
**Pull request by codingbandit**
September updates, also removes known vulnerabilities from projects.
Fixes #31, #29, #25, #26, #27, #28

### CODE REVIEW

**Comment by codingbandit** [comment]
The PR for the lodash can be closed (without merge) as this content update removes lodash completely.

File: Hands-on lab/Before the HOL.md [hunk 1]
```diff
@@ -99,81 +101,79 @@ In this exercise, you will set up your environment you will use for the rest of
 
 In this task you will perform a deployment of a the VM that will be used during this lab.
 
-1.  Open a new private or incognito tab in your web browser, and then connect to <https://github.com/solliancenet/LABVM>. 
+1.  Open a new private or incognito tab in your web browser, and then connect to <https://github.com/microsoft/MCW-IoT-and-the-Smart-City/Hands-on lab/Lab-files/LABVM>.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL.md]: @timahenning @codingbandit 
This link is broken, please fix.

File: Hands-on lab/HOL step-by-step - IoT and the Smart City.md [hunk 2]
```diff
@@ -1428,50 +1544,52 @@ In this task, you will create a Stream Analytics job that filters vehicle teleme
...
+    ![In the Upload input data blade, the browse button is highlighted and the sample-vehicle-telemetry.json file is displayed in the textbox.](images/Hands-onlabstep-by-step-IoTandtheSmartCityimages/media/image104.png 'Select sample input file')
 
-    ![In the Upload test data blade, the browse field is set to the previously defined file.](images/Hands-onlabstep-by-step-IoTandtheSmartCityimages/media/image104.png 'Upload test data blade')
+24. Select **OK**.
 
-26. Select **OK**.
+25. Select **Test query** in the toolbar above the query.
 
-27. Select **Test** in the toolbar above the query.
+    ![In the Query blade, the Test query button is selected.](images/Hands-onlabstep-by-step-IoTandtheSmartCityimages/media/image105.png 'Query blade', 'Test the analytics query with the sample data')
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - IoT and the Smart City.md]: @timahenning @codingbandit 
This image does not resolve in View, please review and fix.

File: Hands-on lab/Before the HOL.md (hunk 1 above)
Comment (codingbandit) [reply Hands-on lab/Before the HOL.md]: Fixed

File: Hands-on lab/HOL step-by-step - IoT and the Smart City.md (hunk 2 above)
Comment (codingbandit) [reply Hands-on lab/HOL step-by-step - IoT and the Smart City.md]: fixed

**Pull request merged**